06_install_ocp.sh

```diff
@@ -5,4 +5,8 @@ set -e
 source common.sh
 source ocp_install_env.sh
 
-$GOPATH/src/github.com/openshift/installer/bin/openshift-install --log-level=debug create cluster --dir ocp
+if [ "$1" = "install-config" ] ; then
+    $GOPATH/src/github.com/openshift/installer/bin/openshift-install --log-level=debug create install-config --dir ocp
+else
+    $GOPATH/src/github.com/openshift/installer/bin/openshift-install --log-level=debug create cluster --dir ocp
```

Comment:
what about we have something like: 

$GOPATH/src/github.com/openshift/installer/bin/openshift-install --log-level=debug create ${1:-cluster} --dir ocp

I've had the need of also generating `ignition-configs` and `manifests` so, it'd be useful to just do `06_install_ocp.sh ignition-configs`

(we may want to rename this script to `06_run_ocp.sh`

Reply:
sounds good.  I updated this.  Made the "create" part an arg, too